components/TopBarProfileMenu.js

```diff
@@ -38,12 +41,69 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { profileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={profileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
     </ListItem>
   );
 };
+UserMenuLinkEntry.propTypes = {
+  profileMenuLink: PropTypes.bool,
+};
+
+const MyAccount = props => {
+  const { LoggedInUser, logout } = props;
+
+  return (
+    <Box as="ul" p={3} my={2}>
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
+      >
+        {({ data, loading }) =>
+          loading === false && data && data.memberInvitations && data.memberInvitations.length > 0 ? (
+            <UserMenuLinkEntry profileMenuLink href="/member-invitations">
+              <FormattedMessage
```

Comment:
The lang files must be updated since you are internationalizing the strings. Can do easily by running, `npm run langs:update`. 🐨